torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1063,8 +1063,10 @@ def simplify(self, expr: "sympy.Expr") -> "sympy.Expr":
             div_replacements = {}
             for atom in expr.atoms(FloorDiv):
                 base, divisor = atom.args
-                if self.replace(base % divisor) in self.divisible:
-                    div_replacements[atom] = sympy.floor(base / divisor)
+                if self.replace(base % divisor) in self.divisible and (
+                    isinstance(base, sympy.Integer) and isinstance(divisor, sympy.Integer)
+                ):
```

Comment:
I don't think it matters that I check the types second because self.divisible is a set